{{cookiecutter.github_repository}}/settings/production.py

```diff
@@ -66,7 +66,8 @@
 SESSION_COOKIE_SECURE = False
 SESSION_COOKIE_HTTPONLY = True
 SECURE_SSL_REDIRECT = env.bool("DJANGO_SECURE_SSL_REDIRECT", default=True)
-
+# Uncomment this if nginx is sending a header for us and we have to enforce https at django layer
+# SECURE_PROXY_SSL_HEADER = ('X-Forwarded-Proto', 'https')
```

Comment:
This things has already been taken care-off, you might wanna remove it from here.